test/parallel/test-cluster-dgram-reuse.js

```diff
@@ -36,5 +36,9 @@ function close() {
     cluster.worker.disconnect();
 }
 
-for (let i = 0; i < 2; i++)
-  dgram.createSocket({ type: 'udp4', reuseAddr: true }).bind(common.PORT, next);
+const socket = dgram.createSocket({type: 'udp4', reuseAddr: true})
+  .bind(0, common.mustCall(() => {
+    const socket2 = socket();
```

Comment:
I was under the impression the calling socket() was the same as dgram.createSocket since I declared socket as a const. I updated the code here 41292b7